### DISCUSSION TITLE
OBS crashes on updating from v28 to 29 inside the app

### DISCUSSION THREAD

**Discussion by bearcatjamboree** [obsproject/obs-studio#8435] [General]
Is anyone from the development team aware of this issue?

This was reported on the forum at the following thread: 

https://obsproject.com/forum/threads/obs-crashes-on-updating-from-v28-to-29-inside-the-app.163099/

I tried digging through the issues here but was unable to find anything.  

I wanted to point out that this appears to happen on Ventura 13.1 as well.  It crashes during the update and then keeps crashing every time you try to launch OBS afterwards.

I had to delete OBS 29 and revert back to OBS 28.1.2 and I'm afraid to try it again, because it caused my entire OS crash at one point.

**Comment by bearcatjamboree** [comment]
To be clear, I'm NOT the person that posted the crash report and log to the forum.  I just haven't seen a response in 2 months so I decided to ask here...

**Comment by RytoEX** [comment]
This looks like a crash somewhere in our updater code. You can safely update OBS by installing OBS Studio 29 normally instead of using the in-app updater. We have significantly reworked our updater implementation for the next release, so it's possible that this would be resolved in future releases.

**Comment by bearcatjamboree** [comment]
I downloaded and installed it without using the updater, and it's hung up doin nothing.

**Reply by RytoEX** [reply]
We do not use GitHub Discussions for application support. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance. If it's determined that the behavior you're experiencing could be a bug, you may be asked to file a GitHub Issue.

Thank you for your understanding.
